hail/python/hailtop/hailctl/dev/deploy/cli.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, deploy_config=None):
         self._session: Optional[httpx.ClientSession] = None
 
     async def __aenter__(self):
-        headers = service_auth_headers(self._deploy_config, 'ci')
+        headers = await hail_credentials().auth_headers()
```

Comment:
What happened to `ci` here?

Reply:
The service argument was basically unused. As long as I've been here all services lived in the same namespace for a given environment but maybe this is an artifact of some time before where that wasn't the case?